stabilizer/src/main/java/com/hazelcast/stabilizer/coordinator/Coordinator.java

```diff
@@ -372,6 +375,32 @@ private void echo(String msg) {
         log.info(msg);
     }
 
+    private void copyUploadDirectoryToAgents() throws IOException {
+        log.info("WORKING DIRECTORY:" + WORKING_DIRECTORY.getAbsolutePath());
```

Comment:
I believe I already commented on this line. But why is this in caps? And the working directory already is obvious because that is the directory the user is in. No need to print it; certainly not when you are dealing with uploads.